lib/pbench/server/cache_manager.py

```diff
@@ -808,16 +798,23 @@ def get_inventory(self, path: str) -> Optional[JSONOBJECT]:
             Dictionary with file info and file stream
         """
         if not path:
-            info = {
+            return {
                 "name": self.tarball_path.name,
                 "type": CacheType.FILE,
                 "stream": Inventory(self.tarball_path.open("rb")),
             }
```

Comment:
Do we need to be concerned by the fact that this `Inventory` instance has no lock?

Reply:
There's nothing to lock, since the tarball isn't in "cache".